doc/radosgw/s3select.rst

```diff
@@ -0,0 +1,268 @@
+===============
+ Ceph s3 select 
+===============
+
+.. contents::
+
+Overview
+--------
+
+    | The purpose of **s3 select** engine is to create an efficient pipe between user client to storage node (the engine should be close as possible to storage).
+    | It enables the user to define the exact portion of data should be received by his side.
```

Comment:
it enables selection of a restricted subset of (structured) data stored in an S3 object using an SQL-like syntax